lib/assessment.rb

```diff
@@ -13,28 +16,31 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (5..10).inject { |sum, n| sum + n }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
-  end
+    a = [7, 5, 9, 2, 0, 7, 9, 4, 2, 0]
+    p a.slice_when {|i, j| i.even? != j.even? }.to_a
+
```

Comment:
No, sorry - this is not correct.

First, we want to be operating on the parameter `numbers` that we've been given.

Second, we need to use an enumerable method to determine if at least some elements in our collection pass a particular test. Can you think of one that would behave in that way?